usaspending_api/references/models.py

```diff
@@ -620,3 +621,27 @@ def get_default_fields(path=None):
             "website_address",
             "objectives",
         ]
+
+
+class Definition(models.Model):
+    id = models.AutoField(primary_key=True)
+    term = models.TextField(unique=True, db_index=True, blank=False, null=False)
+    data_act_term = models.TextField(blank=True, null=True)
+    plain = models.TextField(unique=True)
+    official = models.TextField(blank=True, null=True)
+    slug = models.SlugField(max_length=500, null=True)
+
+    def save(self, *arg, **kwarg):
+        self.slug = slugify(self.term)
+        return super(Definition, self).save(*arg, **kwarg)
+
+
+class DefinitionResource(models.Model):
+    id = models.AutoField(primary_key=True)
+    title = models.TextField()
+    url = models.TextField(null=False, blank=False)
+    definition = models.ForeignKey(Definition, related_name='references')
```

Comment:
Will setting the related_name to `references` cause any issues or confusion because the application name is also `references`? It's probably fine, but just wanted to make sure!